Bottom: b8bc527aa9eec4dad0a1914345469775acb33830
Top:    7837535e55c03671b7605152cf3974d3f6c621a7
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2009-10-20 23:29:49 +0100

Remove "=" shortcut to the djopt() thingy which keeps eating my design

ARR!! how much work have I lost to this key binding!









---

diff --git a/src/gpcb-menu.res b/src/gpcb-menu.res
index a952f0c..bc7deb3 100644
--- a/src/gpcb-menu.res
+++ b/src/gpcb-menu.res
@@ -326,13 +326,11 @@ MainMenu =
    {"Rip up all auto-routed tracks" RipUp(All)}
    -
    {"Optimize routed tracks"
-    {"Auto-Optimize" djopt(auto)  a={"Shift-=" "Shift<Key>="}}
     {"Debumpify" djopt(debumpify) }
     {"Unjaggy" djopt(unjaggy) }
     {"Vianudge" djopt(vianudge) }
     {"Viatrim" djopt(viatrim) }
     {"Ortho pull" djopt(orthopull) }
-    {"Simple optimization" djopt(simple)  a={"=" "<Key>="}}
     {"Miter" djopt(miter) }
     {"Puller" a={"Y" "<Key>y"} Puller() }
     {"Global Puller"
